packages/lib/gpc/test/gpc.spec.ts

```diff
@@ -1,5 +1,11 @@
 import { ProtoPODGPC } from "@pcd/gpcircuits";
-import { POD, PODCryptographicValue, PODValue, PODValueTuple } from "@pcd/pod";
+import {
```

Comment:
I don't see any test cases covering equality checks between real and virtual entries.  Given the odd one-directional way equality comparison is defined, there should be test cases to cover equality defined real->virtual, virtual->real, and virtual->virtual, in addition to the existing tests of real->real.